ios/ReactNativeCharts/RNBarLineChartViewBase.swift

```diff
@@ -6,6 +6,7 @@
 import Foundation
 import Charts
 import SwiftyJSON
+import React
```

Comment:
It's hard to know if this is going to fly upstream. I think it's the right thing to do because all of the React stuff should be in it's own module, but it really depends how you set up your project.